plugins/jobtype/src/azkaban/jobtype/HadoopPigJob.java

```diff
@@ -112,6 +113,11 @@ public void run() throws Exception {
       Props props = new Props();
       props.putAll(getJobProps());
       props.putAll(getSysProps());
+      String additionalNamenodes =
```

Comment:
This code is the same in multiple classes. It would be great to reduce the code duplication.